src/main/kotlin/racing/scene/Scene.kt

```diff
@@ -0,0 +1,9 @@
+package racing.scene
+
+interface Scene {
+
+    fun hasNext(): Boolean
+    fun before() {}
+    fun update()
+    fun after() {}
```

Comment:
```suggestion
    fun before() 
    fun update()
    fun after()
```
불필요한 중괄호는 없애주시면 좋아요.

Reply:
InputScene의 경우 before와 after를 사용하지 않지만 불필요하게 구현하는 문제점을 알려주셔서 기본 메서드를 표현한 중괄호입니당!.. 처음 큰 추상화 인터페이스 Scene을 StartScene, UpdatableScene, DestroyScene과 같이 인터페이스를 세부적으로 나눠봤지만 GameEngine이 요구하는 라이프 사이클을 만족하지 못했습니다 ㅠ..